app/views/govuk_publishing_components/components/_tabs.html.erb

```diff
@@ -1,5 +1,9 @@
 <%
   tabs ||= []
+  no_panel_border ||= false
+  panel_css_classes = %w(govuk-tabs__panel)
+  panel_css_classes << "gem-tabs__panel--no-border" if no_panel_border
```

Comment:
why gem-tabs rather than gem-c-tabs ?

Reply:
To show the styles are being introduced at the gem level

Reply:
I think I need you to explain this to me in person, I thought that's what gem-c-tabs (or gem-c-*) did.

Reply:
Ah, you're right, I'm missing the `-c-`. My bad, reading between the lines. Google spot, will update!

Reply:
👍